runtimes/protocol/getConfigurationFromServer.ts

```diff
@@ -0,0 +1,13 @@
+import { AutoParameterStructuresProtocolRequestType, LSPAny } from './lsp'
+
+export const getConfigurationFromServerRequestType = new AutoParameterStructuresProtocolRequestType<
+    GetConfigurationFromServerParams,
+    LSPAny,
+    never,
+    void,
```

Comment:
Should server be able to return errors?

Reply:
tbh I couldn't think of a reason why servers should return errors in the config scenario, do you have any scenario in mind?